content/vi/docs/concepts/architecture/cloud-controller.md

```diff
@@ -0,0 +1,241 @@
+---
+title: Các khác niệm nền tảng của Cloud Controller Manager
+content_template: templates/concept
+weight: 40
+---
+
+{{% capture overview %}}
+
+Khái niệm Cloud Controller Manager (để tránh nhầm lẫn với bản binary build cùng tên) được định nghĩa để cho phép các bên cung cấp dịch vụ cloud và bộ core của Kubernetes được phát triển độc lập. Bộ CCM chạy đồng thời với những thành phần khác thuộc máy chủ của một cụm như Controller Manager của Kubernetes, API server, và scheduler. Nó cũng có thể đóng vai trò như một addon cho Kubernetes.
+
+Cloud Controller Manager này được thiết kế dựa trên cơ chế plugin nhằm cho phép các bên Cloud Provider có thể tích hợp với Kubernetes một cách dễ dàng thông qua các plugin này. Đã có những bản kế hoạch được thiết kế sẵn nhằm mục đích hỗ trợ những cloud provider thay đổi từ mô hình cũ sang mô hình mới đi chung với CCM.
+
+Tài liệu này thảo luận về những khái niệm đằng sau một bộ CCM và đưa ra những chi tiết về chức năng liên quan của nó.
+
+Dưới đây là kiến trúc của một cụm Kubernetes khi không đi cùng với Cloud Controller Manager:
+
+![Pre CCM Kube Arch](/images/docs/pre-ccm-arch.png)
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Thiết kế
+
+Trong sơ đồ trên, Kubernetes và nhà cung cấp dịch vụ cloud được tích hợp thông qua một số thành phần sau:
+
+- Kubelet
+- Kubernetes Controller Manager
+- Kubernetes API server
+
+Bộ CCM hợp nhất tất cả các logic phụ thuộc trên một nền tàng Cloud từ 3 thành phần trên để tạo thành một điểm tích hợp duy nhất với hệ thống cloud này. Sơ đồ kiến trúc khi đi kèm với bộ CCM sẽ trở thành:
```

Comment:
If you wish, you can also localize the diagram:
localize the English word “connector”, and possibly annotate “cloud” with a localized term.

Reply:
@sftim Thank for your suggestion. But it should be kept "connector" or "cloud" as the original. Because of these original words will be more meaningful and easier to understand than the translation.

Reply:
done anh. I agree on keeping it as original as possible